packages/enhanced_gradients/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+ - **DOCS**(enhanced_gradients): expand readme and the example app.
```

Comment:
Since it's the changelog only for `enhanced_gradients`, I don't think specifying the name here is necessary

Reply:
Done